boards/arm/thingy53_nrf5340/thingy53_nrf5340_common.dts

```diff
@@ -59,6 +59,12 @@
 		};
 	};
 
+	npm1100_force_pwm_mode: npm1100_force_pwm_mode {
+		compatible = "nordic,thingy53-npm1100-force-pwm-mode";
```

Comment:
Why not use "regulator-fixed-sync" instead? Would it bring too much overhead?

Reply:
I thought that making new specific bunding would more clearly show what does this pin do. I can change it to   regulator-fixed-sync if you think it would be better solution.